python/py-pytest-mpl/Portfile

```diff
@@ -1,39 +1,61 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           python 1.0
 PortGroup           github 1.0
+PortGroup           python 1.0
 
-github.setup        matplotlib pytest-mpl 0.9 v
-name                py-${name}
+github.setup        matplotlib pytest-mpl 0.11 v
+revision            0
+name                py-${github.project}
 categories-append   devel
+
 license             BSD
+maintainers         nomaintainer
 platforms           darwin
-supported_archs     noarch
-maintainers         {aronnax @lpsinger} openmaintainer
 
-description         pytest plugin to for testing figures output from Matplotlib
-long_description    This is a plugin to faciliate image comparison for \
-                    Matplotlib figures in pytest. Matplotlib includes a number \
-                    of test utilities and decorators, but these are geared \
-                    towards the nose testing framework. Pytest-mpl makes it \
-                    easy to compare figures produced by tests to reference \
-                    images when using pytest.
+description         pytest plugin to help with testing figures output\
```

Comment:
I'm not entirely sure that the longer description is necessary.

Reply:
This commit dropped. Please see above.